### PR TITLE
mysql: respect `my.cnf` parameter using PyMySQL library

### LINKED ISSUE #7518: Netdata not working with MySQL because SHOW USER STATISTICS MariaDB only command
Netdata is not working with MySQL, only MariaDB.  This is because this command SHOW USER_STATISTICS in this file  only works in MariaDB.

Problem code is https://github.com/netdata/netdata/blob/bfae683664b488df91940a81760459db2b18983c/collectors/python.d.plugin/mysql/mysql.chart.py

My install is http://walkercodetutorials.codes:19999

### PR DESCRIPTION
**Pull request by anirudhdggl**
##### Summary
Fixes #7518

##### Component Name
MySQLService.py

##### Test Plan
Have original users who raised the issue verify it

##### Additional Information
As mentioned in one of the issue comment, the requisite code block responsible for the issue has been removed so that no one faces this issue again

### CODE REVIEW

File: collectors/python.d.plugin/python_modules/bases/FrameworkServices/MySQLService.py [hunk 1]
```diff
@@ -50,12 +50,6 @@ def get_connection_properties(conf, extra_conf):
             elif conf.get('host'):
                 properties['host'] = conf['host']
                 properties['port'] = int(conf.get('port', 3306))
-            elif conf.get('my.cnf'):
-                if MySQLdb.__name__ == 'pymysql':
-                    # TODO: this is probablt wrong, it depends on version
-                    self.error('"my.cnf" parsing is not working for pymysql')
-                else:
-                    properties['read_default_file'] = conf['my.cnf']
```
Comment (ilyam8) [inline collectors/python.d.plugin/python_modules/bases/FrameworkServices/MySQLService.py:58]: i think we shouldnt delete 

```python
elif conf.get('my.cnf'):
    properties['read_default_file'] = conf['my.cnf']
```

we want to get rid of 

```python
if MySQLdb.__name__ == 'pymysql':	
    # TODO: this is probablt wrong, it depends on version	
    self.error('"my.cnf" parsing is not working for pymysql')
```

Comment (anirudhdggl) [reply collectors/python.d.plugin/python_modules/bases/FrameworkServices/MySQLService.py:58]: Updated the pull request

**Comment by ilyam8** [comment]
@anirudhdggl we need you to sign the CLA to get this PR merged

**Comment by anirudhdggl** [comment]
Where can I sign the CLA?

**Comment by ilyam8** [comment]
> Have original users who raised the issue verify it

I guess this is not hard requirement. Indeed old versions of pymysql didnt support loading from `my.cnf`, but it no longer a problem. I think it is safe to merge.

**Comment by ilyam8** [comment]
i see [pymysql](https://github.com/PyMySQL/PyMySQL) has `read_default_file` parameter

https://github.com/PyMySQL/PyMySQL/blob/c3c87a7e773dbb09def0b081c70dd55fe83b9633/pymysql/connections.py#L139-L140

**Pull request merged**